src/core/seq.js

```
@@ -0,0 +1,124 @@
+  if (equalWidth && !fmt) {
+    const width = strs.reduce((w, x) => Math.max(w, x.length), 0);
+    strs = strs.map(x => x.padStart(width, '0'));
+  }
+  // no output if sequence is empty
+  const newEnv = Object.assign({}, env, { '?': 0 });
+  if (strs.length === 0) {
+    return { stdout: '', stderr: '', env: newEnv };
+  }
+  // join with separator and trailing newline
+  const out = strs.join(sep) + '\n';
+  return { stdout: out, stderr: '', env: newEnv };
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)